tests/lib/names.sh

```diff
@@ -1,8 +1,4 @@
 #!/bin/bash
 
-gadget_name=$(snap list | sed -n 's/^\(pc\|pi[23]\|dragonboard\|cm3\) .*/\1/p')
-kernel_name=$gadget_name-kernel
-
-if [ "$kernel_name" = "pi3-kernel" ] || [ "$kernel_name" = "cm3-kernel" ]; then
-    kernel_name=pi2-kernel
-fi
+gadget_name=$(snap list | grep 'gadget$' | awk '{ print $1 }')
```

Comment:
Nitpick, we could look for `-gadget$` and `-kernel$` respectively.

Reply:
I don't think we want that, there's no '-' in the 'Notes' column that we are matching here. Here's an example of what's matched:
pc               16.04-0.8      9     canonical            gadget
pc-kernel        4.4.0-112.135  102   canonical            kernel